tests/php/ORM/HierarchyTest.php

```diff
@@ -682,4 +684,15 @@ public function testDefaultParent(string $class, ?string $defaultParentConfig, ?
 
         $this->assertSame($expected, $obj->defaultParent());
     }
+
+    /**
+     * Tests that HTML added by an extension is not escaped
+     */
+    public function testGetTreeTitleExtension()
+    {
+        Group::add_extension(TestTreeTitleExtension::class);
+        $group = new Group();
+        $group->Title = 'My group';
+        $this->assertSame('<i>My group</i>', $group->getTreeTitle());
```

Comment:
```suggestion
        $group->Title = '<i>My group</i>';
        $this->assertSame('<i>&lt;i&gt;My group&lt;/i&gt;</i>', $group->getTreeTitle());
```
Validating that any HTML in the title itself is still correctly escaped.

Reply:
Updated